### ISSUE TITLE
[Builder] clean up UFO dir not to true by default anymore

### ISSUE THREAD

**Issue by RosaWagner** [googlefonts/gftools#455]
Builder has this option:
``cleanUp``: Whether or not to remove temporary files. Defaults to ``true``.

Temp directories containing UFO are not removed by default at the end of the building process anymore, and it doesn't yell "complete" anymore also. Even if I set up the parameter manually in the config.yaml, they don't get cleaned up although the build works and all.

These directories are supposed to remain only if there was an error message.

This is Rosario as a test case:
[Rosario.zip](https://github.com/googlefonts/gftools/files/7814442/Rosario.zip)

**Comment by RosaWagner** [comment]
I started to see that it does that cause the builder stops at the moment of applying autohinting on italic ttf statics. So italic statics ttf are not exported, and the builder stops with no output message.

**Comment by RosaWagner** [comment]
This issue is messing up lots of automations for building statics upstream

**Comment by simoncozens** [comment]
The autohinting problem is the segmentation fault for Mac M1?

**Comment by RosaWagner** [comment]
Yes, possible. That package is super practical but we regularly have issues with it. We should remove the python dependency and apply autohint with normal ttfautohint with a warning to users to brew install it imo

**Comment by m4rc1e** [comment]
If we drop ttfautohint-py, the unified font repository will also need updating. ttfauothint-py is handy since it includes a wheel for all platforms that we target. By dropping it to just fix the mac m1 issue, we'll need to also system install it for linux etc.

**Comment by simoncozens** [comment]
I seem to remember it was something to do with the error handler callback.
